corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -1839,7 +1839,12 @@ def test_user_data_profile_removal(self):
     def test_uncategorized_data(self):
         self._test_uncategorized_data(is_web_upload=True)
 
+    @patch('corehq.apps.user_importer.importer.domain_has_privilege', lambda x, y: True)
```

Comment:
There's also a `@privilege_enabled` test decorator that you might be interested in